### ISSUE TITLE
fl_getinfo fails

### ISSUE THREAD

**Issue by paoloviviani** [alpha-unito/gam#38]
Ubuntu 17.10 with libfabric 1.4.0 installed from apt.

fl_getinfo fails when performing trivial operations in gam (possibly before doing anything, when creating the context).

```
gam_unit_test: ../3rdparty/gam/include/gam/links_implementations/fl_common.hpp:63: void gam::fl_getinfo(fi_info**, const char*, const char*, uint64_t, fi_ep_type, uint64_t): Assertion `!ret' failed.
```

**Comment by droccom** [comment]
That should by fixed by #40.

**Comment by paoloviviani** [comment]
Fixed
